doc/user/connect/cli.md

```diff
@@ -20,9 +21,17 @@ Detail | Info
 
 Tool | Description | Install
 -----|-------------|--------
-`mzcli` | Materialize adaptation of `pgcli` | [GitHub](https://github.com/MaterializeInc/mzcli#quick-start)
-`psql` | Vanilla PostgreSQL CLI tool. Useful for executing queries from scripts using `-c`. | `libpq` or `postgresql-client`
-`pgcli` | CLI tool with auto-completion & syntax highlighting | [`pgcli` documentation](https://www.pgcli.com/install)
+`mzcli` | Materialize-specific CLI | [GitHub](https://github.com/MaterializeInc/mzcli#quick-start)
+`psql` | Vanilla PostgreSQL CLI tool | `postgresql` or `postgresql-client`
+
+{{< warning >}}
+Not all features of `psql` are supported by Materialize.
+{{< /warning >}}
+
+Other tools built for PostgreSQL can often be made to work with Materialize with
+minor modifications, but are unlikely to work out of the box.
+[File a GitHub issue](https://github.com/MaterializeInc/materialize/issues/new?labels=C-feature&template=feature.md)
```

Comment:
Can you move this to the same line as the preceding sentence?

Reply:
It renders all as one paragraph, if that's your concern? 
![image](https://user-images.githubusercontent.com/882976/80250636-3f870c00-8642-11ea-840f-f0629f77bb1d.png)

I was just trying to avoid the massively long line in the raw markdown.

Reply:
Ah you're right. Sorry!

Reply:
No all good!